src/plone/exportimport/importers/relations.py

```diff
@@ -13,4 +14,10 @@ def do_import(self) -> dict:
             return f"{self.__class__.__name__}: No data to import"
         logger.debug(f"- Relations: Read {len(data)} from {self.filepath}")
         results = utils.set_relations(data)
+        for item in results:
+            # Reindex relations objects in case they contain
+            #  `preview_image_link` fields to update the image scales
+            api.content.get(UID=item["to_uuid"]).reindexObject()
+            api.content.get(UID=item["from_uuid"]).reindexObject()
```

Comment:
@sneridagh @ericof We just need to update the catalog metadata, not the indexes, so you can save time by reindexing a single index (for example UID) instead of the default which is all indexes.

Reply:
@davisagli how do you refresh only metadata, one not tied to any index? I was under the assumption that you can only do that by reindexing the whole object.

Reply:
@sneridagh just pick one index randomly, it will reindex that one index but at least it won't do the others. e.g. `obj.reindexObject(idxs=["UID"])`